dclab/kde/base.py

```diff
@@ -200,8 +202,8 @@ def get_contour_lines(self, xax="area_um", yax="deform", xacc=None,
             normalize=False)
         contours = []
         # Normalize levels to [0, 1]
-        qlevels = np.array(levels) / density.max()
-        for level in qlevels:
+        qlev = np.array(levels) / density.max()
```

Comment:
I literally meant replace `level` (not `qlevels`) with `qlev`.

The reason is that we have to explicitly distinguish between density levels and quantile levels. When I read `level` in the code, I think it's a density level. But `level` here is a quantile level, so I would change it to `qlev`.

Reply:
Sorry, I overlooked this.